src/nvim/msgpack_rpc/channel.c

```diff
@@ -761,6 +767,14 @@ static Channel *register_channel(ChannelType type, uint64_t id,
   kv_init(rv->call_stack);
   kv_init(rv->delayed_notifications);
   pmap_put(uint64_t)(channels, rv->id, rv);
+
+  ILOG("registered ch %" PRIu64 " (%s): %s", rv->id,
```

Comment:
"register" is some weird internal jargon, how about "opened rpc channel". (when we have proper introspection later, we can just dump the dict with some nice formatting)